### DISCUSSION TITLE
Budibase App Deployment Best Practices

### DISCUSSION THREAD

**Discussion by fritzwijaya** [Budibase/budibase#8172] [Help] [answered]
Hi budibase community,

Currently I am evaluating the budibase platform for our internal tools platform. Its looks promising and solid.
But, I have questions regarding the best practice to do deployment for the apps that we build on budibase.

What is the best practice to do deployment for the apps? I am thinking to do deployment from git repository, to keep track the changes of the apps, and enabling CI/CD for each budibase apps that we are develop. Is there any docs that I could refer to achieve this?

Thanks. Regards

**Comment by melohagan** [comment, marked as answer]
Hey @fritzwijaya 

Thank you for your interest in Budibase!

Version control is currently being worked on: https://github.com/Budibase/budibase/issues/4360 
It will be included in the Team plan: https://budibase.com/pricing/
